storage/remote/read_handler.go

```diff
@@ -57,8 +57,8 @@ func NewReadHandler(logger log.Logger, r prometheus.Registerer, queryable storag
 		marshalPool:               &sync.Pool{},
 
 		queries: prometheus.NewGauge(prometheus.GaugeOpts{
-			Namespace: "prometheus",
-			Subsystem: "api", // TODO: changes to storage in Prometheus 3.0.
+			Namespace: namespace,
+			Subsystem: "remote_read_handler",
```

Comment:
why not `storage` as you said in PR title?

Reply:
https://github.com/prometheus/prometheus/pull/14642#discussion_r1716712415 :)